doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -4035,10 +4026,9 @@ \subsubsection{Outputs}\label{outputs-10-005}
   HVAC,Average,Boiler Part Load Ratio
 \end{itemize}
 
-~ One of the following blocks will be applicable based on fuel type:
+\noindent One of the following blocks will be applicable based on fuel type:
```

Comment:
This whole section should be condensed with a single set using a generic name "Boiler <Fuel Type> Rate". For an example, see how this is done for Coil:Heating:Fuel. And I see that's already how the definitions are done below.